prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -293,7 +268,15 @@ def smart_charging(
 
```

Comment:
What is  emfacvmt`? Should it be in the `const` module

Reply:
`emfacvmt` is being replaced with the `Regional_scaling_factors` csv files.  Ultimately we'll be removing `emfacvmt`, I assume in a future PR